.travis.yml

```diff
@@ -18,17 +18,20 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4
 
 services:
   - mysql
 
 # Different stages in order to only deploy when everything succeeds
 stages:
   - test
+  - name: compatibility
+    if: branch = master
   - e2e-testing
 
 env:
-  - WP_VERSION=latest WC_VERSION=4.3.0
+  - WP_VERSION=latest WC_VERSION=4.5.0
```

Comment:
There is already 4.6.0, so maybe worth updating before shipping.

Reply:
Yep, we can do this. Updating.